sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_enums.py

```diff
@@ -0,0 +1,77 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from enum import Enum
+
+
+class KeyVaultRoleScope(str, Enum):
+    """Collection of well known role scopes. This list is not exhaustive"""
+
+    global_value = "/"  #: use this if you want role assignments to apply to everything on the resource
+
+    keys_value = "/keys"  #: use this if you want role assignments to apply to all keys
+
+
+class DataActionPermission(str, Enum):
```

Comment:
/cc @christothes that's a good idea for all languages as well.

Reply:
Hm... since these would be used to populate a `KeyVaultPermission` object and they do describe actions, I agree that something like `KeyVaultDataAction` might make more sense.